sbin/common/common.sh

```diff
@@ -129,7 +129,14 @@ function setBootJdk() {
 
     # shellcheck disable=SC2046,SC2230
     if [[ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "darwin" ]]; then
+      set +e
       BUILD_CONFIG[JDK_BOOT_DIR]="$(/usr/libexec/java_home)"
+      set -e
+      local returnCode=$?
```

Comment:
Isn't this going to use the return code of `set -e` which is always zero?

Reply:
Good point, let me fix that